docs/PacketFence_Developers_Guide.asciidoc

```diff
@@ -178,6 +178,15 @@ Navigate to menu:File[Import>Import csv] to import a csv file.
 
 a|Navigate to menu:File[Import>Import csv] to import a csv file.
 
+|GUI button and menu (non experimental)
```

Comment:
non-experimental

Reply:
We might want to write "GUI buttons and menus...." - if we want to use the plural form everywhere.

Reply:
I use singular form because we use previously:
```
GUI Button
[..]
GUI Menu
```